dev-haskell/apply-refact/files/apply-refact-0.8.2.1-use-builtin-ghc-boot-th.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/apply-refact.cabal b/apply-refact.cabal
+index c470110..95a1094 100644
+--- a/apply-refact.cabal
++++ b/apply-refact.cabal
+@@ -30,7 +30,7 @@ library
+                    , Refact.Apply
+                    , Refact.Fixity
+   other-modules: Refact.Internal
+-  GHC-Options: -Wall
++  GHC-Options: -Wall -package ghc-boot-th
```

Comment:
This is not an upstream patch. it is needed due to [this line](https://github.com/mpickering/apply-refact/blob/master/apply-refact.cabal#L105) in the upstream cabal file, which explicitly depends on an external `ghc-boot-th`.

I've seen other packages add a flag to allow this to be configurable, but in our case, since our ghc package ships with ghc-boot-th, this will allow the tests to run properly.

Reply:
I still think it's not a correct way to add a missing dependency. `GHC-Options:` usually does not change visible packages and only enabled extra warnings or optimizations. Depenencies should go to respective `Build-depends:`.

Also note: there is no difference between `builtin` and `external` `ghc-boot-th` they are all packages from `Cabal` resolver.

Reply:
hrm, maybe I need to put it back in `build-depends`, but remove it from `DEPEND` in the ebuild.

I ran into issues with the version of `ghc-boot-th` that portage found versus what the package was expecting. I'll dig into this a bit more, but I think we may be stuck here if we want to run this test suite.

Reply:
It turns that this patch was only needed whenever I had included ebuilds for `ghc-boot` and `ghc-boot-th`. This was when I was still confused about which packages ship with ghc and which don't.

This patch is no longer needed and has been removed from the PR.